pkg/wipeout/tasks.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2022 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package wipeout
+
+import (
+	"context"
+
+	"github.com/apigee/registry/connection"
+	"github.com/apigee/registry/log"
+	"github.com/apigee/registry/rpc"
+)
+
+// DeleteApiTask deletes a specified API.
+type DeleteApiTask struct {
```

Comment:
Should these Tasks be just wipeout-related? Or would such Tasks (and processing of Tasks) be an interesting abstraction to make available in general?

Reply:
@seaneganx suggested general tasks [in this comment](https://github.com/apigee/registry/pull/631#discussion_r910381339) and a preceeding issue. I'm open to that, which might mean moving these elsewhere.